docker-entrypoint.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash 
+
+set -e
+
+service postgresql start 
+msfdb init
```

Comment:
init only needs to run once, would prefer that in a layer. What's the benefit of having `postgresql start` in here versus `CMD service postgresql start && /bin/bash`?

Reply:
run once per image or once per run of a container?

generally for these kinds of situations, using an ENTRYPOINT script is preferred as it lets you do more complicated setup items if necessary. I actually initially had more logic in the script w.r.t trapping signals and doing cleanup, but i ended up removing that because somehow postgres processes were getting reaped (unexpectedly good)

Reply:
also fwiw, systemd is never going to work in this container. they're largely orthogonal things (systemd vs containers) so directly doing anything with `systemctl` is pointless, they'll always fail

Reply:
Once per image - it sets up the msf database and initial schema.

Reply:
ok i moved it back to a `RUN` in the Dockerfile, but i kept the entrypoint script to do `service postgresql start` and `msfdb start` just so metasploit can run its DB checks